one.mixin.messenger.desktop.yml

```diff
@@ -0,0 +1,51 @@
+id: one.mixin.messenger.desktop
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: mixin_desktop
+rename-desktop-file: mixin_desktop.desktop
+rename-icon: mixin_desktop
+rename-appdata-file: mixin_desktop.metainfo.xml
```

Comment:
since all the files are here instead of coming from upstream (the latter is prefered), you might as well name them properly from the get go.